### DISCUSSION TITLE
Is there an updated version of debloat.bat that removes the Asus optimization service?

### DISCUSSION THREAD

**Discussion by Metalogic999** [seerge/g-helper#444] [Q&A] [answered]
Is there an updated version of debloat.bat that removes the Asus optimization service, as sounds like this isn't necessary (and may conflict with G-Helper) for the latest G-Helper releases? Thanks

**Comment by seerge** [comment, marked as answer]
@Metalogic999 no, as coupe of users (like on old TUF laptops) seem to report that keyhandling doesn't work w/o optimization doesn't work for them. So first I want to figure out if it works for everyone or not - and then update debloater (actually i plan to add it in the app as a button)

**Reply by Metalogic999** [reply]
Okay, understood. If I did want to manually add a line to remove the Asus optimization service in debloater.bat, what would I need to add? Thanks

**Reply by seerge** [reply]
"ASUSOptimization" :)

**Reply by Metalogic999** [reply]
Thanks, like this, right?:

sc STOP ASUSOptimization
sc config  ASUSOptimization start= disabled
